ARKitProject/Utilities.swift

```diff
@@ -436,9 +436,9 @@ extension ARSCNView {
 		let maxAngleInDeg = min(coneOpeningAngleInDegrees, 360) / 2
 		let maxAngle = ((maxAngleInDeg / 180) * Float.pi)
 		
-		let points = features.points
+		let points = features.__points
```

Comment:
why the double underscores?  does something change on https://developer.apple.com/documentation/arkit/arframe/2887449-rawfeaturepoints?